velox/exec/Task.h

```diff
@@ -344,6 +361,11 @@ class Task : public std::enable_shared_from_this<Task> {
       uint32_t splitGroupId,
       const std::vector<core::PlanNodeId>& planNodeIds);
 
+  // Adds Customized Hash Joinbridge for all the specified plan node IDs.
```

Comment:
typos: "Adds custom join bridges for the specified plan nodes."

Reply:
Fixed